app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/index.tsx

```diff
@@ -16,221 +16,37 @@
 
 import React, { useEffect, useReducer } from 'react';
 import T from 'i18n-react';
-import styled from 'styled-components';
 import { createContextConnect } from 'components/Replicator/Create';
 import { Map } from 'immutable';
 import Button from '@material-ui/core/Button';
-import CloseIcon from '@material-ui/icons/Close';
-import IconButton from '@material-ui/core/IconButton';
 import { getCurrentNamespace } from 'services/NamespaceStore';
 import { MyReplicatorApi } from 'api/replicator';
-import Checkbox from '@material-ui/core/Checkbox';
 import LoadingSVG from 'components/LoadingSVG';
 import Heading, { HeadingTypes } from 'components/Heading';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
-import Radio from '@material-ui/core/Radio';
-import RadioGroup from '@material-ui/core/RadioGroup';
 import SearchBox from 'components/Replicator/Create/Content/SearchBox';
-import { IColumnsList, IColumnImmutable, ITableInfo } from 'components/Replicator/types';
-import { useDebounce } from 'services/helpers-2';
+import { IColumnImmutable, ITableInfo } from 'components/Replicator/types';
+import { useDebounce } from 'services/react/customHooks/useDebounce';
+
+import { ISelectColumnsProps, ReplicateSelect } from './types';
+import { reducer, initialState } from './reducer';
+import {
+  ActionButtons,
+  Backdrop,
+  ButtonWithMarginRight,
+  GridWrapper,
+  Header,
+  LoadingContainer,
+  Root,
+  RadioContainer,
+  StyledCheckbox,
+  StyledRadio,
+  StyledRadioGroup,
+  SubtitleContainer,
+} from './styledComponents';
```

Comment:
nit: I might name this "styles" to differentiate it from the upstream styled components